src/test/ui/issue-47377.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+fn main(){
+    let b = "hello";
```

Comment:
This file doesn't use tabs. You need to change the indentation of these two lines to use tabs and then add a line `// ignore-tidy-tab` to get tidy passing, like done in `src/test/ui/codemap_tests/tab_2.rs`.

Reply:
@est31 I changed it to `~^ ERROR E0369
}`. Do I still need //ignore-tidy-tab?

Reply:
I think `ignore-tidy-tab` is so that the `tidy` check for the project doesn't block the merge of the PR, while keeping the tabs in the code so that we avoid regressions in the handling of them. I believe your editor might be "helpfully" replacing tabs with spaces.

Reply:
Yeah it was. Can you have a look now?